### PR TITLE
indexer: set endblock to the actual end block for an election

### PR DESCRIPTION
**Pull request by p4u**
If the election is ended manually, the indexer stores as endblock the current height.

Furthermore, add a new boolean value to let the API caller know that the election was ended manually.

### CODE REVIEW

**Comment by p4u** [comment]
ping @mvdan

File: api/api_types.go [hunk 1]
```diff
@@ -25,14 +25,15 @@ type OrganizationList struct {
...
-	Results        [][]*types.BigInt `json:"result,omitempty"`
+	ElectionID      types.HexBytes    `json:"electionId" `
+	OrganizationID  types.HexBytes    `json:"organizationId" `
+	Status          string            `json:"status"`
+	StartDate       time.Time         `json:"startDate"`
+	EndDate         time.Time         `json:"endDate"`
+	VoteCount       uint64            `json:"voteCount"`
+	FinalResults    bool              `json:"finalResults"`
+	Results         [][]*types.BigInt `json:"result,omitempty"`
+	ManuallyStopped bool              `json:"manuallyStopped"`
```
Comment (mvdan) [inline api/api_types.go]: ManuallyEnded, to be consistent with EndDate and EndBlock?

File: vochain/indexer/process.go [hunk 2]
```diff
@@ -202,6 +207,7 @@ func (idx *Indexer) newEmptyProcess(pid []byte) error {
 		EntityID:          nonNullBytes(eid),
 		StartBlock:        int64(p.StartBlock),
 		EndBlock:          int64(p.BlockCount + p.StartBlock),
```
Comment (mvdan) [inline vochain/indexer/process.go:209]: should we still insert EndBlock when creating a new process? it's StartBlock+BlockCount, so until the process is actually ended (either automatically or manually), it's not really useful at all.

perhaps it's best to always have it there for consistency, but wanted to flag this anyway.

**Review by mvdan — APPROVED** [review]

File: vochain/indexer/process.go (hunk 2 above)
Comment (p4u) [reply vochain/indexer/process.go:209]: yes, for consistency, it makes sense to set it up when creating the process. Thanks for flagging.

**Pull request merged**